src/tailwind.css

```
@@ -0,0 +1,4 @@
+@layer theme, base, antd, components, utilities;
+@import "tailwindcss";
+
+@source "./src";
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)